config.yaml

```diff
@@ -123,7 +123,7 @@ communications:
 
   # Settings for Mattermost
   mattermost:
-    enabled: true
+    enabled: false
     url: 'MATTERMOST_SERVER_URL'                # e.g https://example.com:9243
     token: 'MATTERMOST_TOKEN'
     team: 'MATTERMOST_TEAM'
```

Comment:
We should add one liner description for these fields

Reply:
Made the required changes.